github/GitRelease.py

```diff
@@ -165,6 +165,7 @@ def update_release(
         message: str,
         draft: bool = False,
         prerelease: bool = False,
+        make_latest: Opt[bool] = NotSet,
```

Comment:
Adding this here may break user code, can you add this at the end of the argument list?